### PR TITLE
Establish Standards to Support Web Access to SBOM Data Paper

### PR DESCRIPTION
**Pull request by goneall**
This paper proposes an approach to establish a set of standard for web access to SBOM data based on previous standards activities.

My apologies for getting this in late - I needed a bit of extra time to review the approach with some of the folks involved in the standards.

### CODE REVIEW

**Comment by jkowalleck** [comment]
really would love to have this topic part of the agenda.
see also: https://github.com/w3c/secure-the-web-forward-workshop/blob/main/papers/sbom-for-web-frontends.md#retrieval-of-sbom

File: papers/web-standards-for-sbom-access.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+By [Gary O'Neall](#author).
+
+
+## Description
+Establishing a common vocabulary and protocol for accessing SBOM information over the public internet would ease integration into
+various software build and deployment systems thereby improving overall software supply chain security.
+
+Although this is a very ambitious goal, there is already a lot of standardization we can leverage for this purpose.
+
+There is an existing [IETF draft proposel](https://datatracker.ietf.org/doc/html/draft-ietf-opsawg-sbom-access)
```
Comment (tidoust) [inline papers/web-standards-for-sbom-access.md]:
```suggestion
There is an existing [IETF draft proposal](https://datatracker.ietf.org/doc/html/draft-ietf-opsawg-sbom-access)
```

File: papers/web-standards-for-sbom-access.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+to extend the Manufacturers Usage Description ([RFC 8520](https://tools.ietf.org/html/rfc8520)) to discover and retrieve SBOM information.
+Although this proposal is focused on devices at the end of the supply chain, aspects of the model and approach could be leveraged for 
+more general SBOM discovery scenarios.
+
+For the SBOM vocabulary, the SPDX standard had defined terms for supporting use cases involving SBOMs as well as many other supply chain relevant use cases.
+
+The SPDX standard is a formal ISO standard ([ISO/IEC 5962:2021](https://www.iso.org/standard/81870.html)) as well an
```
Comment (tidoust) [inline papers/web-standards-for-sbom-access.md]:
```suggestion
The SPDX standard is a formal ISO standard ([ISO/IEC 5962:2021](https://www.iso.org/standard/81870.html)) as well as an 
```

File: papers/web-standards-for-sbom-access.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+SPDX is supported and maintained by a large group of contributors representing tooling providers, SBOM producers and SBOM consumers.
+
+The upcoming SPDX 3.0 has some very specific enhancements for making the SBOM information more web accessible.
+
+Leveraging an existing well-defined vocabulary that supports standards ranging from JSON Schemas to full RDF Ontologies can accelerate
+the adoption of internet accessible SBOM data and improve overall software supply chain security.
+
+
+## Links
+- [IETF draft proposel for SBOM Access](https://datatracker.ietf.org/doc/html/draft-ietf-opsawg-sbom-access)
```
Comment (tidoust) [inline papers/web-standards-for-sbom-access.md]:
```suggestion
- [IETF draft proposal for SBOM Access](https://datatracker.ietf.org/doc/html/draft-ietf-opsawg-sbom-access)
```

File: papers/web-standards-for-sbom-access.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+various software build and deployment systems thereby improving overall software supply chain security.
+
+Although this is a very ambitious goal, there is already a lot of standardization we can leverage for this purpose.
+
+There is an existing [IETF draft proposel](https://datatracker.ietf.org/doc/html/draft-ietf-opsawg-sbom-access)
+to extend the Manufacturers Usage Description ([RFC 8520](https://tools.ietf.org/html/rfc8520)) to discover and retrieve SBOM information.
+Although this proposal is focused on devices at the end of the supply chain, aspects of the model and approach could be leveraged for 
+more general SBOM discovery scenarios.
+
+For the SBOM vocabulary, the SPDX standard had defined terms for supporting use cases involving SBOMs as well as many other supply chain relevant use cases.
```
Comment (tidoust) [inline papers/web-standards-for-sbom-access.md]:
```suggestion
For the SBOM vocabulary, the SPDX standard defines terms for supporting use cases involving SBOMs as well as many other supply chain relevant use cases.
```

**Review by tidoust — APPROVED** [review]
Thanks, @goneall! I confirm acceptance of the position statement. I'll add it to the published list.

**Pull request merged**